### ISSUE TITLE
Missing CANCER_TYPE_DETAILED in tcga studies?

### ISSUE THREAD

**Issue by pieterlukasse** [cBioPortal/datahub#17]
In cbioportal.org we now see the pancancer histogram view for BRCA study:

![image](https://cloud.githubusercontent.com/assets/2900303/20831022/838d5df4-b883-11e6-957a-184ed72f7b24.png)

However, this does not seem to appear in the study loaded from datahub? Is the `CANCER_TYPE_DETAILED` field missing or empty in the datahub tcga studies?

**Comment by pieterlukasse** [comment]
Notify: @n1zea144 @zheins : just something I noticed today, but I could be wrong about whether it is missing in datahub (maybe you recently added it). Please let me know.

**Comment by sandertan** [comment]
When the study is downloaded from cbioportal.org, the datahub version is downloaded which misses CANCER_TYPE_DETAILED in sample/patient data. Would be cool to have it included!

**Comment by zheins** [comment]
@pieterlukasse @sandertan - We generate that via our internal importing pipelines. I'll add it to the files.

**Comment by sandertan** [comment]
@zheins Could you check that the correct attribute is used to fill this column? This is from `brca_tcga`, it seems to miss the information from the public portal:
<img width="878" alt="screen shot 2017-04-12 at 16 33 51" src="https://cloud.githubusercontent.com/assets/9624990/24963172/211a8e58-1f9e-11e7-8198-6661b2b8a664.png">

**Comment by pieterlukasse** [comment]
@sandertan @zheins I checked this on the public portal again and what @sandertan observes there seems to be correct. The only difference is that "undefined" is called "NA" in study view. When you click "Customize histogram" you will see the other cancer types. They don't appear on the plot because the samples labelled with these cancer types just don't have mutations in TP53. I found another gene where they do have mutations, and it works as expected. See screenshot below:

![image](https://user-images.githubusercontent.com/2900303/28366603-cacee552-6c8d-11e7-91ee-d6ae32abc481.png)
